src/tabletraits.jl

```diff
@@ -0,0 +1,42 @@
+Tables.isrowtable(::Type{<:DESolution}) = true
+Tables.columns(x::DESolution) = Tables.columntable(Tables.rows(x))
+
+struct DESolutionRows{T, U}
+    names::Vector{Symbol}
+    types::Vector{Type}
+    lookup::Dict{Symbol, Int}
+    t::T
+    u::U
+end
+
+DESolutionRows(names, types, t, u) = DESolutionRows(names, types, Dict(nm => i for (i, nm) in enumerate(names)), t, u)
+
+Base.length(x::DESolutionRows) = length(x.u)
+Base.eltype(x::DESolutionRows{T, U}) where {T, U} = DESolutionRow{eltype(T), eltype(U)}
```

Comment:
```suggestion
Base.eltype(::Type{DESolutionRows{T, U}}) where {T, U} = DESolutionRow{eltype(T), eltype(U)}
```

Reply:
@quinnj does this work for the Tables.jl interface?

Reply:
The generic fallback ensures that there are no problems, regardless if you call it with types or instances, doesn't it?

Reply:
I didn't know there's a generic fallback there.